lib/puppet/parser/scope.rb

```diff
@@ -638,9 +638,9 @@ def handle_not_found(class_name, variable_name, position, reason = nil)
     unless Puppet[:strict_variables]
       # Do not issue warning if strict variables are on, as an error will be raised by variable_not_found
       location = if position[:lineproc]
-                   " at #{position[:lineproc].call}"
+                   Puppet::Util::Errors.error_location_with_space("", position[:lineproc])
```

Comment:
Converting file to use error_location_with_space, which returns the translated location string for the output.

Reply:
I missed adding the '.call', I'll fix that.

I had noticed that we don't set this anywhere, and was wondering if we could delete this code since we don't use it.